stock_indicators/_cstypes/decimal.py

```diff
@@ -18,7 +18,11 @@ class Decimal:
         2.5
     """
     def __new__(cls, decimal) -> CsDecimal:
-        return CsDecimal.Parse(str(decimal))
+        try:
+            return CsDecimal.Parse(str(decimal))
+        except CsFormatException as e:
+            raise ValueError("This may be a locale settings issue in your environment.\n"
```

Comment:
> You may be using numeric data that is incompatible with your locale environment settings.  For example, you may be using decimal points instead of commas.

Reply:
Thanks :)